### DISCUSSION TITLE
Welcome to librdkafka Discussions!

### DISCUSSION THREAD

**Discussion by edenhill** [edenhill/librdkafka#3188] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about librdkafka here in discussions rather than opening a Github issue.
  * Share ideas.
  * Engage with other community members, help out, be friendly.

**Comment by ShivaKothuri** [comment]
Does LIBRDKAFKA support LDAP Authentication?
If yes, can you please share the example?
